pkg/core/logger/logger.go

```diff
@@ -140,7 +140,7 @@ func makeInternalFileOutput(cfg *Config) (zapcore.Core, error) {
 	// defaultCfg is used to set the defaults for the file rotation of the internal logging
 	// these settings cannot be changed by a user configuration
 	defaultCfg := logp.DefaultConfig(logp.DefaultEnvironment)
-	filename := filepath.Join(paths.Logs(), cfg.Beat)
+	filename := filepath.Join(paths.Home(), DefaultLogDirectory, cfg.Beat)
```

Comment:
change line 129 as well
`cfg.Files.Path = filepath.Join(paths.Home(), DefaultLogDirectory)` may be conflicting
not sure why you have no logs though, i tested it and had logs at both location,
internal output is not listening to logging config aside from log level.
non internal is altering behavior on other logging settings

Reply:
Sorry, but what is the change that should be made to line 129?  And what's the reason for it?

> internal output is not listening to logging config aside from log level.
> non internal is altering behavior on other logging settings.

This is not strictly related to the change you're suggesting above but I'm asking more for my own understanding because I have to admit, I'm a bit confused by the terminology:

What is the purpose of the internal log output?  When is it supposed to be used and when is it not supposed to be used?  That is, when creating a logger using the `logger.New` or `logger.NewFromConfig` constructors, when is it appropriate to pass the `logInternal` argument as `true` and when should it be `false`?

Reply:
> Sorry, but what is the change that should be made to line 129? And what's the reason for it?

@michalpristas Thanks for making the change via a3524db285e04ae832bf979059fe0d7d062d461b.  I'd like to understand the reason for it and also the other question I asked about the internal log output but that can wait until after this PR is merged.